doc/backing-store-crd.md

```diff
@@ -80,3 +92,209 @@ status:
       lastTime: "2019-06-04T13:05:35.473Z"
       troubleshooting: "https://github.com/noobaa/noobaa-core/wiki/Backing-store-finalizer-troubleshooting"
 ```
+
+# Example
+
+Here is an example of BackingStore CR when running different BackingStore types:
```

Comment:
Minor suggestion:

Here are some example BackingStore CRs for various backing-store types: